src/MPMD.hpp

```diff
@@ -3,6 +3,8 @@
 #include <string>
 #include <mpi.h>
 #include <assert.h>
+#include <map> //JM
+#include <string.h> //JM
```

Comment:
Why `<string.h>` when you have `<string>` ?
I have to migrate the changes to https://github.com/CFD-GO/RemoteForceInterface